src/test/scala/cz/alenkacz/db/pasquared/ConfigParserTest.scala

```diff
@@ -0,0 +1,36 @@
+package cz.alenkacz.db.pasquared
+
+import com.github.mauricio.async.db.Configuration
+import com.github.mauricio.async.db.pool.PoolConfiguration
+import com.typesafe.config.ConfigFactory
+import cz.alenkacz.db.pasquared.ConfigParser.PostgresConfiguration
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.concurrent.duration._
+import scala.concurrent.duration.Duration
+import scala.util.Success
+
+@RunWith(classOf[JUnitRunner])
+class ConfigParserTest extends FlatSpec with Matchers {
+  private val config = ConfigFactory.load("test.conf")
```

Comment:
We can have more tests with different settings so I would prefer to nest current config.